scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
 
-
     options : dict, optional
-        A dictionary of solver options. All methods accept the following
-        options:
-
-            init : 2d-array (default = 'barycenter')
-                The algorithm may be sensitive to the initial permutation
-                matrix (or search position) chosen due to the possibility
-                of several local minima within the feasible region.
-                With only 1 initialization, a barycenter init will
-                likely return a more accurate permutation.
-
-                Choosing several random initializations (through
-                `init_weight` and `init_n`) as opposed to the non-informative
-                barycenter will likely result in a more accurate result at
-                the cost of higher runtime.
-
-                The initial position chosen:
-
-                "barycenter" : the non-informative "flat doubly stochastic
-                matrix," :math:`J=1*1^T /n` , i.e the barycenter of the
-                feasible region (where :math:`n` is the number of nodes and
-                :math:`1` is a ``(n, 1)`` array of ones).
-
-                If an ndarray is passed, it should have the same shape as
-                `cost_matrix` and `dist_matrix`, and its rows and columns
-                must sum to 1 (doubly stochastic).
-            init_weight : float, positive in range [0,1] (Default = None)
-                Allows the user to specify the amount of random perturbation
-                from the starting position of `init`
-                At each initialization, the initial permutation matrix
-                is some random point near :math:`J`, defined as
-                :math:`(\alpha J + (1- \alpha) K`, where :math:`\alpha`
-                is `init_weight` and :math:`K` is some random doubly
-                stochastic matrix.
-            init_n : int, positive (default = 1)
-                Number of random initializations of the starting
-                permutation matrix that the FAQ algorithm will undergo.
+        A dictionary of solver options.
+
+            init_J : 2d-array or "barycenter" (default = "barycenter")
+                The initial (guess) permutation matrix or search "position"
+                :math:`J`.
+
+                :math:`J` need not be a proper permutation matrix;
+                however, it must have the same shape as `cost_matrix` and
+                `dist_matrix`, and it must be doubly stochastic: each of its
+                rows and columns must sum to 1.
+
+                If unspecified or ``"barycenter"``, the non-informative "flat
+                doubly stochastic matrix" :math:`1*1^T/n`, where :math:`n`
+                is the number of nodes and :math:`1` is a :math:`n \times 1`
+                array of ones, is used. This is the "barycenter" of the
+                search space of doubly-stochastic matrices.
+            init_weight : float in range [0, 1] (default = 1)
```

Comment:
```suggestion
            init_alpha : float in range [0, 1] (default = 1)
```
I would be comfortable calling this `init_alpha`, but leaving it alone is OK, too.

Reply:
During FW, there is an `alpha` variable which specifies the "step size" (the degree in which we move the permutation matrix in the search direction), so I think it would be better to keep it as weight to avoid confusion.

Reply:
OK